run-example.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash -e
 EXAMPLE_MAP=(
+  'uring:io.netty.example.uring.EchoIOUringServer'
```

Comment:
Should be `iouring` instead of `uring` in my opinion.

Reply:
maybe we should use `io_uring`